src/ResourceManager/Resources/Commands.Resources.Test/Providers/GetAzureProviderCmdletTests.cs

```diff
@@ -153,7 +153,7 @@ public void GetsResourceProviderTests()
             };
             var pagableLocations = new Page<Location>();
             pagableLocations.SetItemValue<Location>(locationList);
-            var locationsResult = new AzureOperationResponse<IPage<Location>>()
+            var locationsResult = new AzureOperationResponse<IEnumerable<Location>>()
```

Comment:
As long as it's not a breaking change here, it's OK, but IMO, you want to fix this in the sdk, you don't want to go from pageange to not pageable back to pageable